pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -465,10 +466,10 @@ def create_bondwire(
         end_position :  list
             List of ``[x, y, z]`` coordinates for the ending position
             of the bond pad.
-        h1 : float, optional
+        h1 : float|str optional
```

Comment:
@MaxJPRey Are you using a new method for documenting the datatype and whether it's optional? If so, would someone please educate me? Thanks!